speedtest/check.sh

```diff
@@ -14,20 +14,20 @@ touch "$OUTPUT_FILE"
 
 # ERROR CODES
 ERR_NO_SPEEDTEST=1
-ERR_NO_SERVERS=2
-ERR_SPEEDTEST=3
 
-which speedtest >/dev/null 2>&1
-if [ $? != 0 ]; then
+if ! command -v speedtest >/dev/null 2>&1; then
   echo "Speedtest CLI not found. Aborting!" >> "$LOG_FILE"
-  echo -e "PATH:\t$PATH" >> "$LOG_FILE"
+  printf "PATH:\t%s" "$PATH" >> "$LOG_FILE"
   exit $ERR_NO_SPEEDTEST
 fi
 
 echo "server_id,server_name,location,time,distance_km,latency_ms,download_bits,upload_bits,unknown_9,client_ip" >> "$OUTPUT_FILE"
 
 oldIFS=$IFS
-IFS=$'\n'
+IFS="
+"
```

Comment:
What's the reason for this change?

Reply:
https://github.com/koalaman/shellcheck/wiki/SC2039

I think I was just reading some random stackoverflow for it at the time. Feel free to disregard.

Reply:
I think it's meant to be

IFS=$(echo -e '\n')

Reply:
I think maybe this is equivalent.
IFS="$(printf '%b_' '\n')"; IFS="${IFS%_}" 

Tbh, I don't really know why I did this PR. I think I just like seeing some of the weird stuff come from shellcheck tbh.